production/helm/loki/Chart.yaml

```diff
@@ -4,7 +4,7 @@ name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
 appVersion: 2.6.1
-version: 3.2.2
+version: 3.2.3
```

Comment:
sorry, just slipped in `3.3.0` under this, need to bump this to `3.3.1` now

Reply:
Updated the version